docs/installation/integrations/next-js.md

```diff
@@ -0,0 +1,204 @@
+
+## Preparing a build
+
+First, we will use the [online builder](https://ckeditor.com/ckeditor-5/online-builder/). It is a web UI that lets you create a custom build of CKEditor&nbsp;5 and download the code as a zip package.
+
+The online builder is a powerful tool that lets you effortlessly create a rich text editor tailored to your needs. With the online builder, you can choose the desired editor type and plugins, configure the toolbar, and choose the UI language for your editor.
+
+You can learn more about creating custom builds of CKEditor&nbsp;5 with the online builder in the {@link installation/getting-started/quick-start-other#creating-custom-builds-with-online-builder Customized installation} guide.
+
+## Setting up the project
+
+This guide assumes you will use the official CLI tool for Next.js &ndash; `create-next-app`. Refer to the [Next.js documentation](https://nextjs.org/docs/pages/api-reference/create-next-app) to learn how to set up your project.
+
+The CLI will ask you some questions. Depending on your choices, the folder structure in your project may differ.
+
+```plain
+What is your project named?  next-ckeditor-integration
+Would you like to use TypeScript?  No / Yes
+Would you like to use ESLint?  No / Yes
+Would you like to use Tailwind CSS?  No / Yes
+Would you like to use `src/` directory?  No / Yes
+Would you like to use App Router? (recommended)  No / Yes
+Would you like to customize the default import alias (@/*)?  No / Yes
+```
+
+### Page Router
+
+If you chose the Page Router, your folder structure should look similar to the one below.
+
+```plain
+├── pages
+│   ├── api
+│      └── hello.js
+│   ├── _app.js
+│   ├── _document.js
+│   └── index.js
+├── public
+│   ├── favicon.ico
+│   ├── next.svg
+│   └── vercel.svg
+├── styles
+│   ├── globals.css
+│   └── Home.module.css
+├── jsconfig.json
+├── next.config.js
+├── package.json
+└── ...
+```
+
+### App Router
+
+If you chose the App Router, your folder structure should look similar to the one below.
+
+```plain
+├── app
+│   ├── favicon.ico
+│   ├── globals.css
+│   ├── layout.js
+│   ├── page.js
+│   └── page.module.css
+├── public
+│   ├── next.svg
+│   └── vercel.svg
+├── jsconfig.json
+├── next.config.js
+├── package.json
+└── ...
+```
+
+## Integrating the build in your Next.js project
+
+With the Next.js setup done, let's move to the actual integration. First, put your unzipped custom build at the project root in the `ckeditor5` folder. Then, add it as a local dependency using the yarn command:
```

Comment:
I would somehow use the file structures you have above to point out where the `ckeditor5` folder should go, to better reference this visually.